src/homepage/components/kaiUiComponents/Header/Header.scss

```diff
@@ -0,0 +1,32 @@
+
+
+// $kai-header-height: 2.8rem;
+
+.kai-heade {
+  
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  border-bottom: 1px solid #E4E4E4;
+
+  height:30px;
+
+
+ 
+}
+ .h1style{
+  
+  display: inline-flex;
```

Comment:
![Screenshot 2020-06-03 at 11 18 38 AM](https://user-images.githubusercontent.com/8564518/83600346-02406300-a58c-11ea-8b01-d946f93e5372.png)
Inline-flex might not be supported in kaios